scripts/build_docs.sh

```diff
@@ -178,4 +177,4 @@ else
     echo "Check the logs in $logs_dir, and the documentation build output in $docs_build_output."
 fi
 
-cd "$working_dir" && if $is_sourced; then return $docs_exit_code; else exit $docs_exit_code; fi
```

Comment:
Agreed. At the same time, it makes it easy to forget setting is_sourced every time before it is used (or after invoking other scripts that also define it)... Since that tripped me up a couple of times, I opted to make it a one liner. If there is a strong preference, I can replace the oneliner with two lines everywhere.